docs/pages/Project.md

```diff
@@ -34,4 +34,31 @@ The source directory must contain a directory structure representing the namespa
 The `module` type is meant for usage with non-UI5 resources like third-party libraries. Their path mapping can be configured freely. During a build, their resources are copied without modifications.
 
 
+## Build Output Style
+
+The Build Output Style offers developers control over their project's build output folder. Namespaces like `sap.m` or `sap.ui.core` can be streamlined, producing a more concise and flat output. For example, a resource like `/resources/sap/m/RangeSlider.js` will be transformed into `./RangeSlider.js`. And vice versa, applications that are built by default with `Flat` output, can leverage the namespaces they have. 
+
+**Note**: Build Output Style is applied only over root project's output folder structure. Any dependencies included in the build would remain with their `Default` output style.
+
+In the table below are the available combinations between project type & build output style.
+
+| Requested Output Style / Project Type | Resulting Style |
+|---|---|
+| **Default** | |
+| `application` | Root project is written `Flat`-style, dependencies in their respective `Default`-style  |
```

Comment:
what about writing ` dependencies in their respective `Default`-style` as a foot note?

Reply:
Done